.eslintrc.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
-  plugins: ["import", "prettier"],
-  extends: ["plugin:import/recommended", "plugin:import/typescript", "plugin:prettier/recommended", "react-app"],
+  plugins: ["import"],
+  extends: [
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:prettier/recommended",
+    "react-app",
+    "prettier",
+  ],
```

Comment:
I'm just extra glad to see the line breaks again. I couldn't even read it before! I see what you did here.